pkg/kubernetes/kubernetes.go

```diff
@@ -159,6 +189,19 @@ func (k *Kubernetes) Derived(ctx context.Context) *Kubernetes {
 	if err != nil {
 		return k
 	}
-	derived.Helm = helm.NewHelm(derived)
 	return derived
 }
+
+func (k *kubernetes) CacheInvalidate() {
```

Comment:
Why do we need invalidating the cache?. Doesn't MCP server interact with the same cluster throughout its lifetime?. Stdio would require this. But maybe when stdio is used, we fully use a different path.

Reply:
We should check that sync, the test doesn't validate that behavior (I think). (I'll add a TODO)

Reply:
The test using this is here: https://github.com/manusa/kubernetes-mcp-server/blob/54d564a54e38febc63c56c60b9ccb76501adf55f/pkg/mcp/pods_top_test.go#L10

There's a fake API server behind which does reply. Only one request is performed.
I'll set the TODO and we can check later